fiscalsim_us/data/datasets/cps/cps.py

```diff
@@ -1,354 +1,358 @@
+    # Add pensions and annuities.
+    cps_pensions = person.PNSN_VAL + person.ANN_VAL
+    cps["taxable_pension_income"] = cps_pensions * (
+        p["taxable_pension_fraction"][year]
+    )
+    cps["tax_exempt_pension_income"] = cps_pensions * (
+        1 - p["taxable_pension_fraction"][year]
+    )
+    # Other income (OI_VAL) is a catch-all for all other income sources.
+    # The code for alimony income is 20.
+    cps["alimony_income"] = (person.OI_OFF == 20) * person.OI_VAL
+    # The code for strike benefits is 12.
+    cps["strike_benefits"] = (person.OI_OFF == 12) * person.OI_VAL
+    cps["child_support_received"] = person.CSP_VAL
+    # Assume all public assistance / welfare dollars (PAW_VAL) are TANF.
+    # They could also include General Assistance.
+    cps["tanf_reported"] = person.PAW_VAL
+    cps["ssi_reported"] = person.SSI_VAL
+    cps["pension_contributions"] = person.RETCB_VAL
+    cps["long_term_capital_gains"] = person.CAP_VAL * (
+        p["long_term_capgain_fraction"][year]
+    )
+    cps["short_term_capital_gains"] = person.CAP_VAL * (
+        1 - p["long_term_capgain_fraction"][year]
+    )
+    cps["receives_wic"] = person.WICYN == 1
+    cps["veterans_benefits"] = person.VET_VAL
+    cps["workers_compensation"] = person.WC_VAL
+    # Disability income has multiple sources and values split across two pairs
+    # of variables. Include everything except for worker's compensation
+    # (code 1), which is defined as WC_VAL.
+    WORKERS_COMP_DISABILITY_CODE = 1
+    disability_benefits_1 = person.DIS_VAL1 * (
+        person.DIS_SC1 != WORKERS_COMP_DISABILITY_CODE
+    )
+    disability_benefits_2 = person.DIS_VAL2 * (
+        person.DIS_SC2 != WORKERS_COMP_DISABILITY_CODE
+    )
+    cps["disability_benefits"] = disability_benefits_1 + disability_benefits_2
+    # Expenses.
+    # "What is the annual amount of child support paid?"
+    cps["child_support_expense"] = person.CHSP_VAL
+    cps["health_insurance_premiums"] = person.PHIP_VAL
+    cps["medical_out_of_pocket_expenses"] = person.MOOP
+
+
+def add_spm_variables(cps: h5py.File, spm_unit: DataFrame) -> None:
+    SPM_RENAMES = dict(
+        spm_unit_total_income_reported="SPM_TOTVAL",
+        snap_reported="SPM_SNAPSUB",
+        spm_unit_capped_housing_subsidy_reported="SPM_CAPHOUSESUB",
+        free_school_meals_reported="SPM_SCHLUNCH",
+        spm_unit_energy_subsidy_reported="SPM_ENGVAL",
+        spm_unit_wic_reported="SPM_WICVAL",
+        spm_unit_payroll_tax_reported="SPM_FICA",
+        spm_unit_federal_tax_reported="SPM_FEDTAX",
+        spm_unit_state_tax_reported="SPM_STTAX",
+        spm_unit_work_childcare_expenses="SPM_CAPWKCCXPNS",
+        spm_unit_medical_expenses="SPM_MEDXPNS",
+        spm_unit_spm_threshold="SPM_POVTHRESHOLD",
+        spm_unit_net_income_reported="SPM_RESOURCES",
+        childcare_expenses="SPM_CHILDCAREXPNS",
+    )
+
+    for openfisca_variable, asec_variable in SPM_RENAMES.items():

```

Comment:
@ss7886. These two instances of the name `openfisca_variable` in lines 346 and 347 should be `fiscalsim_variable`.